src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -945,6 +941,23 @@ private StringBuilder getFeedbackSessionResultsForQuestionInCsvFormat(
         return exportBuilder;
     }
 
+    private int getMaxNumberOfResponseComments(List<FeedbackResponseAttributes> allResponses,
+                                               Map<String, List<FeedbackResponseCommentAttributes>> responseComments) {
```

Comment:
Hi @LimeTheCoder, yep that should be fine. Usually, when we say "use 8 spaces", it tends to mean "use an additional 8 spaces as compared to the previous line". The general idea is just to avoid custom indentation.